blockly-core/core/ui/block.js

```diff
@@ -1336,7 +1336,7 @@ Blockly.Block.prototype.setUserVisible = function(userVisible, opt_renderAfterVi
   }
   // Apply to all children recursively
   this.childBlocks_.forEach(function (child) {
-    child.setUserVisible(userVisible);
+    child.setUserVisible(userVisible, opt_renderAfterVisible);
```

Comment:
This was supposed to be passed down recursively. This successfully resolves the Firefox rendering issue [#90225590].

Before:

![image](https://cloud.githubusercontent.com/assets/206973/6719836/d3492512-cd7a-11e4-84cf-668d71cddcb8.png)


After:

![image](https://cloud.githubusercontent.com/assets/206973/6719798/99f453d6-cd7a-11e4-9d54-240cb20a2125.png)